tools/Google.Cloud.Tools.VersionCompat/CecilUtils/ShowExtensions.cs

```diff
@@ -30,6 +30,18 @@ private static string Show(this IList<GenericParameter> ps)
             return "";
         }
 
+        private static string ShowGenArgs(this IList<TypeReference> types)
```

Comment:
Any reason for not using the conditional ?: operator and an expression bodied method here?

Reply:
No reason, done :)